README.md

```diff
@@ -1,9 +1,73 @@
-# autofs #
+# puppet-module-autofs
+
+Manage autofs
+
 ===
 
-Automounter for NFS.
+# Compatibility
+
+This module has been tested to work on the following systems with Puppet v3 and Ruby versions 1.8.7, 1.9.3 and 2.0.0.
+
+  * RedHat EL5 & EL6
+  * Suse 10 & 11
+  * Ubuntu 12.04
+  * Debian 7.7
+
+===
+
+# Parameters
+
+enable
+------
+Boolean value to enable autofs. 
+
+- *Default*: true
+
+mounttimeout
+-------
+Numeric value to set timeout of mounts in seconds.
+
+- *Default*: 300
+
+umountwait
+----------
+Numeric value to try and prevent expire delays when trying to umount from a server
+that is not available.
+
+- *DEFAILT*: undef
+
+browsable
+---------
+String value to set browse mode. Valid values are 'yes', 'YES', 'no' and 'NO'.
+
+- *Default*: 'no'
+
+mounts
+------
+Array value to set mount paths.
```

Comment:
in stdlib 4.6.0 `validate_absolute_path()` now supports arrays, so use that to validate this. I believe the mounts have to be absolute paths right? If so, that should also be mentioned in the README.